views/search_results.erb

```diff
@@ -10,11 +11,11 @@
       </tr>
       <% flights.each do |flight| %>
         <tr>
-          <td><%= Time.parse(flight.takeoff_at).strftime('%I:%M %p') %></td>
-          <td><%= Time.parse(flight.land_at).strftime('%I:%M %p') %></td>
+          <td><%= TimeMath.datetime_for_airport(date, flight.takeoff_at, flight.orig).strftime('%I:%M %p %Z') %></td>
+          <td><%= TimeMath.datetime_for_airport(date, flight.land_at, flight.dest).strftime('%I:%M %p %Z') %></td>
```

Comment:
You're welcome. Yeah looking at that code it would be fun to test when a flight leaves one place before daylight savings change and then lands after daylight savings change. With tests like that templates just get in the way.